projects/schematics/src/add-spartacus/index_spec.ts

```diff
@@ -109,6 +109,34 @@ describe('add-spartacus', () => {
     );
   });
 
+  describe('Verify if target app is standalone', () => {
```

Comment:
nitpick: We don't check whether an app is standalone, but whether app.module.ts exists. Btw. I wouldn't mind if we drop the `describe()` and leave just the `it()`